Furball.Engine/Engine/Timing/AudioStreamTimeSource.cs

```diff
@@ -0,0 +1,11 @@
+using sowelipisona;
+
+namespace Furball.Engine.Engine.Timing {
+    public class AudioStreamTimeSource : ITimeSource {
```

Comment:
really tiny thing but comments would be nice, even if tbh its self explanitory

Reply:
gotta be honest i dont know how to comment things past "this is what we grab the current time from" and "here we set what to grab the time from" which is more comments on the languages syntax then how the code works

Reply:
hmm, fair